app/controllers/api/resources_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  class ResourcesController < ApiApplicationController
+    def show
```

Comment:
Ojo no estaba en el doc de inicio de ciclo, porque no es para MVP, sino para Nice to Have. Por ahora no creo que haga daño.